src/plugins/dataflow/nodes/utilities/view-utilities.ts

```diff
@@ -69,3 +69,9 @@ export function hasFlowIn(node: Node){
 export function getNumDisplayStr(n: number){
   return isNaN(n) ? kEmptyValueString : n.toFixed(3).replace(/\.?0+$/, "");
 }
+
+export function getInsertionOrder(editor: NodeEditor, id: number) {
+  const index = editor.nodes.findIndex((n: Node) => n.id === id);
```

Comment:
It seems like nodes could have name changes based on this code. If I have a dataflow tile, then add a number node, then add a sensor node, I'd expect the sensor node's name to be "Sensor 2". If I delete the number node, I expect it would now be "Sensor 1". Is this acceptable?